cookbook/case_studies/ml_training/house_price_prediction/house_price_predictor.py

```diff
@@ -143,41 +172,44 @@ def split_data(
         ),
     )
 
-
-# %%
-# Task: Generating & Splitting the Data
-# ======================================
-# Call the previously defined helper functions to generate and split the data. Finally, return the DataFrame objects.
+# Now, let's create a NamedTuple that maps variable names (train_data) to data type (DataFrame).
+# This NamedTuple indicates that each variable name would store a DataFrame (train data, validation data, and test data respectively). 
 dataset = typing.NamedTuple(
     "GenerateSplitDataOutputs",
     train_data=pd.DataFrame,
     val_data=pd.DataFrame,
     test_data=pd.DataFrame,
 )
 
-
+# %%
+# Defining a Task to generate and split the dataset
```

Comment:
Can we have simple headings? Training, Evaluation.

This particular task can be a part of the previous heading.